src/main/java/Duke.java

```diff
@@ -1,10 +1,73 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("================================================");
+
+        int i = 0;
+        int marker;
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        do{
+            userInput = in.nextLine();
+
+            if(userInput.startsWith("mark")){
+                marker = Integer.parseInt(userInput.substring(5));
+                tasks[marker-1].markAsDone();
+                System.out.println(tasks[marker-1].printTask());
+            }else if(userInput.startsWith("unmark")){
+                marker = Integer.parseInt(userInput.substring(7));
+                tasks[marker-1].markAsUndone();
+                System.out.println(tasks[marker-1].printTask());
+            }else{
```

Comment:
There are quite a few levels of nesting here - maybe you can place the switch case in a separate method!